securedrop/pretty_bad_protocol/_meta.py

```diff
@@ -296,7 +296,10 @@ def update_path(environment, path):  # type: ignore[no-untyped-def]
                 :param list path: A list of strings to update the PATH with.
                 """
                 log.debug("Updating system path...")
-                os.environ = environment
+                # This assignment is mostly to reset the monkey patch, but it doesn't
+                # actually reset the environment. Leaving it alone since it works as-is
+                # despite being wrong.
+                os.environ = environment  # noqa: B003
```

Comment:
Since this is a partial vendoring anyway, intuitively I flip the burden of proof: we'd have to justify the change, not the non-change.  How about:

```suggestion
                # This assignment doesn't reset the environment, but it does reset the monkey-patch as intended.  Leaving as-is from upstream.
                os.environ = environment  # noqa: B003
```

I defer to you; I won't wordsmith you further.  :-)

Reply:
That works for me, thanks & done.